test-data/src/test/resources/org/openmrs/module/fhir2/providers/LocationWebTest_create.json

```diff
@@ -1,5 +1,20 @@
 {
   "resourceType": "Location",
+  "meta": {
+    "versionId": "1",
+    "lastUpdated": "2021-11-30T08:43:54.105+00:00",
+    "source": "#DNbZg5doCf3wLAkB",
+    "profile": [
+      "http://ihe.net/fhir/StructureDefinition/IHE.mCSD.FacilityLocation"
+    ],
+    "tag": [
+      {
+        "system": "http://facility_registry/mcsd_location",
```

Comment:
It's best we don't include a system here... there's no meaningful system for this. Really, in this case, it should just be "code", I think...